backend/btrixcloud/colls.py

```diff
@@ -526,9 +539,17 @@ async def get_collection_search_values(self, org: Organization):
         names = [name for name in names if name]
         return {"names": names}
 
-    async def get_collection_crawl_ids(self, coll_id: UUID) -> List[str]:
-        """Return list of crawl ids in collection"""
+    async def get_collection_crawl_ids(
+        self, coll_id: UUID, public_or_unlisted_only=False
+    ) -> List[str]:
+        """Return list of crawl ids in collection, including only public collections"""
         crawl_ids = []
+        if public_or_unlisted_only:
+            try:
+                await self.get_collection_raw(coll_id, public_or_unlisted_only)
+            except HTTPException:
+                return []
```

Comment:
This could use a test case

Reply:
Added, should actually be throwing here, nice catch!